action/rlp_tx.go

```diff
@@ -87,14 +85,80 @@ func DecodeRawTx(rawData string, chainID uint32) (tx *types.Transaction, sig []b
 
 // NewEthSigner returns the proper signer for Eth-compatible tx
 func NewEthSigner(txType iotextypes.Encoding, chainID uint32) (types.Signer, error) {
-	// TODO: use proper signer according to tx type
 	switch txType {
 	case iotextypes.Encoding_IOTEX_PROTOBUF:
 		// native tx use same signature format as that of Homestead
 		return types.HomesteadSigner{}, nil
-	case iotextypes.Encoding_ETHEREUM_RLP:
+	case iotextypes.Encoding_ETHEREUM_EIP155:
```

Comment:
iotextypes.Encoding_ETHEREUM_POST_EIP155
iotextypes.Encoding_ETHEREUM_PRE_EIP155?

Reply:
`Encoding_ETHEREUM_UNPROTECTED` is a better name/term, Ethereum call it unprotected tx, and there is `Protected() bool` func

Reply:
there won't be backwards compatibility issue, _EIP155 is an alias of _RLP
```
Encoding_ETHEREUM_EIP155 == Encoding_ETHEREUM_RLP == 1
```